.github/workflows/deploy-gcp-tests.yml

```diff
@@ -89,10 +89,6 @@ on:
 env:
   # where we get the Docker image from
   IMAGE_NAME: zebrad-test
-  GAR_BASE: us-docker.pkg.dev/zealous-zebra/zebra
-  # what kind of Google Cloud instance we want to launch
-  ZONE: us-central1-f
-  MACHINE_TYPE: c2d-standard-16
```

Comment:
This machine type is larger than the machine type in other files, so it needs a different variable

Reply:
It also possibly needs a different zone, in the rare case some machine types are available in different zones, and we can't find a common zone that has both.